.github/workflows/on-tag-push-create-release.yml

```
@@ -0,0 +1,254 @@
+    - name: Download prepared release assets
+      uses: actions/download-artifact@v4
+      with:
+        name: release-assets-${{ needs.common-vars.outputs.tag-name }}
+        path: ${{ env.RELEASE_DIR }}
+
+    - name: Create Release Notes
+      run: |
+        TAG_NAME="${{ needs.common-vars.outputs.tag-name }}"
+        cat > ${{ env.RELEASE_DIR }}/RELEASE_NOTES.md << EOF
+        # PiView Android ${TAG_NAME}
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)